packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -1082,5 +1082,21 @@ describe('build-schema', () => {
         expect(optionalNameSchema.title).to.equal('ProductPartial');
       });
     });
+
+    it('uses custom title when provided by user', () => {
+      @model({title: 'IgnoredTitle'})
+      class TestModel {
+        @property()
+        id: string;
+      }
+
+      const schema = getJsonSchema(TestModel, {
+        title: 'NewTestModel',
```

Comment:
Done in 3d3e79b5.

While improving the tests, I modified my new tests to call `modelToJsonSchema` instead of `getJsonSchema`, because they are not relying on caching behavior of `getJsonSchema`.